### PR TITLE
Fix the incorrect format when describing deployment

### LINKED ISSUE #20862: kubectl describe deployment prints MinReadySeconds as string (should be int)
```
$ kubectl describe -f examples/extensions/deployment.yaml 
Name:			nginx-deployment
...
MinReadySeconds:	%!s(int=0)
...
```

https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/describe.go#L1618 should use `%d` instead. 

cc @kubernetes/kubectl @mqliang

### PR DESCRIPTION
**Pull request by janetkuo**
Fixes #20862 

cc @kubernetes/kubectl @mqliang

### CODE REVIEW

File: pkg/kubectl/describe.go [hunk 1]
```diff
@@ -1615,7 +1615,7 @@ func (dd *DeploymentDescriber) Describe(namespace, name string) (string, error)
 		fmt.Fprintf(out, "Selector:\t%s\n", labels.FormatLabels(d.Spec.Selector))
 		fmt.Fprintf(out, "Replicas:\t%d updated | %d total | %d available | %d unavailable\n", d.Status.UpdatedReplicas, d.Spec.Replicas, d.Status.AvailableReplicas, d.Status.UnavailableReplicas)
 		fmt.Fprintf(out, "StrategyType:\t%s\n", d.Spec.Strategy.Type)
-		fmt.Fprintf(out, "MinReadySeconds:\t%s\n", d.Spec.MinReadySeconds)
+		fmt.Fprintf(out, "MinReadySeconds:\t%d\n", d.Spec.MinReadySeconds)
```
Comment (kargakis) [inline pkg/kubectl/describe.go]: Should we show MinReadySeconds if it is zero? @bgrant0607

**Comment by kargakis** [comment]
One question, LGTM otherwise

File: pkg/kubectl/describe.go (hunk 1 above)
Comment (deads2k) [inline pkg/kubectl/describe.go]: > Should we show MinReadySeconds if it is zero? @bgrant0607

Why wouldn't we?  Do we hide valid values for other describers?

Comment (kargakis) [inline pkg/kubectl/describe.go]: >  Do we hide valid values for other describers?

Not sure. I thought MinReadySeconds could stay unspecified but that's not the case. I am fine with this.

Comment (bgrant0607) [inline pkg/kubectl/describe.go]: Yes, this is fine

**Comment by bgrant0607** [comment]
Lgtm

**Pull request merged**